docs/pages/guides/configuration-management/configuration-management.md

```diff
@@ -7,6 +7,12 @@ type: guides
 category: configuration
 language: en
 abstract: Learn how to use Meshery Designs effectively
+redirect_from:
+  - /tasks/patterns.md
+  - /reference/mesheryctl
```

Comment:
```suggestion

```
This would make https://docs.meshery.io/reference/mesheryctl unreachable, always forwarding to https://docs.meshery.io/guides/configuration-management/working-with-designs

can be seen here: https://65a6b159c7028710af919999--meshery-docs-preview.netlify.app/reference/mesheryctl